.gitignore

```diff
@@ -8,7 +8,6 @@
 .mypy_cache
 .hypothesis
 .tox
-.python-version
```

Comment:
@abhinavsingh do not remove this. It's used by pyenv for saving the user-selected dir-local envs/pythons and mustn't be committed.

Reply:
Ok makes sense, just saw this comment.